lib/bridge/IrcBridge.js

```diff
@@ -560,6 +560,7 @@ IrcBridge.prototype.getThirdPartyProtocol = function(protocol) {
         },
         instances: servers.map((server) => {
             return {
+                network_id: server.domain,
```

Comment:
It's rare, but it can change like when we hop to IPv6 from IPv4. We may want to indirect here (e.g. have a `network_id` value in the config file, but default to `server.domain` if one is not supplied).

This becomes relevant when we inevitably move Moznet to v6 (it's still v4 atm).